Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN npm config set cafile "/app/DOIRootCA2.cer" \
 
 RUN node_modules/bower/bin/bower install --allow-root
 
+RUN nosetests --all-modules --exe
```

Comment:
Still having the same issue-- the build is successful, but it runs 0 tests. I'll do some googling.